gax/src/main/resources/dependencies.propertoes

```diff
@@ -0,0 +1 @@
+version.gax=1.56.0
```

Comment:
Please add newline char.
Please justify the need of copying the file. What harm does including the whole file makes? 
Even if we do the copy, shouldn't it be created only as build artifacts but not included in git (otherwise it is an unnecessary duplication, and confusing source artifacts created during build)?

Reply:
Removed from git.